cmd/kops/update_cluster.go

```diff
@@ -190,6 +205,18 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 	}
 
+	var phaseValidation fi.LifecycleValidationPolicy
```

Comment:
I think we should have a default here, because otherwise I can pass "" and phaseValidation won't be set.  Or we simply don't check  `if c.PhaseValidationPolicy != "" {`